test/test-mnist.rb

```diff
@@ -0,0 +1,104 @@
+class MnistTest < Test::Unit::TestCase
+  include Helper::Sandbox
+
+  def setup_data
+    setup_sandbox
+
+    def @dataset.cache_dir_path
+      @cache_dir_path
+    end
+
+    def @dataset.cache_dir_path=(path)
+      @cache_dir_path = path
+    end
+    @dataset.cache_dir_path = @tmp_dir
+
+    def @dataset.download(output_path, url)
+      directory = (Pathname.new(__dir__) + "tmp").expand_path
+      n_image, image_size_x, image_size_y, label = 10, 28, 28, 1
+
+      Zlib::GzipWriter.open(output_path) do |gz|
+        if output_path.basename.to_s.include?("-images-")
+          image_data = ([2051, n_image]).pack('N2') +
+                       ([image_size_x,image_size_y]).pack('N2') +
+                       ([0] * image_size_x * image_size_y).pack("C*") * n_image
+          gz.puts(image_data)
+        else
+          label_data = ([2049, n_image]).pack('N2') +
+                       ([label] * n_image).pack("C*")
+          gz.puts(label_data)
+        end
+      end
+    end
+  end
+
+  def teardown
+    teardown_sandbox
+  end
+
+  sub_test_case("mnist") do
+
```

Comment:
Can you remove an empty line after `do`?

Reply:
I removed empty line.